### ISSUE TITLE
[FEATURE] Filters for display tables

### ISSUE THREAD

**Issue by PaxITIS** [eduardolat/pgbackweb#31]
Add filters within the various display screens so as to simplify the search.
Four very useful filters for example in the “Executions” screen would be:
- “Status” filter
- “Backup” filter
- “Database” filter
- Filter “Started at”

**Comment by JourneyOver** [comment]
Yes, honestly would love to have some filters on these tables and even possibly a search as well bar as well.

**Comment by proexis-assessoria** [comment]
It is an extremely important feature.
